app/portainer/settings/authentication/ldap/ldap-custom-admin-group/ldap-custom-admin-group.html

```diff
@@ -1,129 +1,127 @@
+            ng-model="config.GroupAttribute"
+            placeholder="member"
+            limited-feature-dir="{{::$ctrl.limitedFeatureId}}"
+            limited-feature-class=" {{ $ctrl.isLimitedFeatureSelfContained && 'limited-be' }}"
+            ng-disabled="{{ $ctrl.isLimitedFeatureSelfContained }}"
+            limited-feature-tabindex="-1"
+          />
+        </div>
+      </div>
+      <div class="form-group">
+        <label for="ldap_admin_group_filter_{{ $index }}" class="col-sm-4 col-md-2 control-label text-left">
+          Group Filter
+          <portainer-tooltip message="'The LDAP search filter used to select group elements, optional.'"></portainer-tooltip>
+        </label>
+        <div class="col-sm-8 col-md-10 vertical-center">
+          <input
+            type="text"
+            class="form-control"
+            id="ldap_admin_group_filter_{{ $index }}"
+            ng-model="config.GroupFilter"
+            placeholder="(objectClass=groupOfNames)"
+            limited-feature-dir="{{::$ctrl.limitedFeatureId}}"
+            limited-feature-class=" {{ $ctrl.isLimitedFeatureSelfContained && 'limited-be' }}"
+            ng-disabled="{{ $ctrl.isLimitedFeatureSelfContained }}"
+            limited-feature-tabindex="-1"
+          />
+          <button class="btn btn-md btn-danger" type="button" ng-click="$ctrl.onRemoveClick($index)" ng-if="$index > 0">
+            <pr-icon icon="'trash-2'" size="'md'"></pr-icon>
+          </button>
+        </div>
+      </div>
     </div>
-  </div>
-</div>
 
-<div class="form-group mt-3">
-  <div class="col-sm-12">
-    <button
-      class="label label-default interactive no-border vertical-center"
-      ng-click="$ctrl.onAddClick()"
-      limited-feature-dir="{{ $ctrl.limitedFeatureId }}"
-      limited-feature-class=" {{ $ctrl.isLimitedFeatureSelfContained && 'limited-be' }}"
-      limited-feature-disabled
-    >
-      <pr-icon icon="'plus-circle'"></pr-icon>
-      add group search configuration
-    </button>
-  </div>
-  <div class="col-sm-12 vertical-center mt-2">
-    <button
-      class="btn btm-sm btn-primary"
-      type="button"
-      ng-click="$ctrl.search()"
-      limited-feature-dir="{{ $ctrl.limitedFeatureId }}"
-      limited-feature-class=" {{ $ctrl.isLimitedFeatureSelfContained && 'limited-be' }}"
-      ng-disabled="{{ $ctrl.isLimitedFeatureSelfContained }}"
-      limited-feature-tabindex="-1"
-    >
-      Fetch Admin Group(s)
-    </button>
-    <span ng-if="$ctrl.groups && $ctrl.groups.length === 0" class="vertical-center ml-6">
-      <pr-icon icon="'alert-triangle'" mode="'warning'"></pr-icon>
-      No groups found
-    </span>
-  </div>
-</div>
+    <div class="form-group mt-3">
+      <div class="col-sm-12">
+        <button disabled class="btn btn-sm btn-light vertical-center !ml-0" ng-click="$ctrl.onAddClick()">
+          <pr-icon icon="'plus'"></pr-icon>
+          Add group search configuration
+        </button>
```

Comment:
same question as other PR

Reply:
This section of the form in both LDAP and AD settings are BE only and don't need the teaser, so disabling is ok